### PR TITLE
[script.litebox] 1.0.0

### PR DESCRIPTION
**Pull request by BADMS**
- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project.
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [script.foo.bar] 1.0.0

### Description
Lightweight decluttered version of script.colorbox (and probably defacto version from now on - no more colorbox).

### CODE REVIEW

**Comment by BADMS** [comment]
Are you guys ok to squash into 1 commit, I can't do it. Or I delete completely and pull again?

File: script.litebox/default.py [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+import xbmcaddon
+import xbmcvfs
+from functools import reduce
+from threading import Thread
+ADDON =             xbmcaddon.Addon()
+ADDON_VERSION =     ADDON.getAddonInfo('version')
+ADDON_LANGUAGE =    ADDON.getLocalizedString
+ADDON_PATH =        ADDON.getAddonInfo('path')
+ADDON_ID =          ADDON.getAddonInfo('id')
+ADDON_DATA_PATH =   os.path.join(xbmcvfs.translatePath("special://profile/addon_data/%s" % ADDON_ID))
```
Comment (ronie) [inline script.litebox/default.py:13]: not a showstopper, but in the future consider using:
xbmcvfs.translatePath(ADDON.getAddonInfo('profile'))

File: script.litebox/resources/lib/utils.py [hunk 2]
```diff
@@ -0,0 +1,468 @@
...
+from urllib.parse import unquote
+import math
+from PIL import Image, ImageOps, ImageEnhance, ImageDraw, ImageStat, ImageFilter
+from .imageoperations import MyGaussianBlur
+from decimal import *
+from threading import Thread
+ADDON =             xbmcaddon.Addon()
+ADDON_ID =          ADDON.getAddonInfo('id')
+ADDON_LANGUAGE =    ADDON.getLocalizedString
+ADDON_DATA_PATH =   os.path.join(xbmcvfs.translatePath("special://profile/addon_data/%s" % ADDON_ID))
```
Comment (ronie) [inline script.litebox/resources/lib/utils.py:18]: same here

**Review by ronie — APPROVED** [review]
looks good to me, thx!

**Comment by BADMS** [comment]
> looks good to me, thx!

Excellent thanks ronie 😷👍

**Pull request merged**